tests/testthat/test_ui.R

```diff
@@ -6,3 +6,56 @@ test_that("test i18n_state", {
   # expect returning shiny tag
   expect_equal(class(i18n_state("a")), "shiny.tag")
 })
+
+test_that("Test usei18n", {
+  # Arrange
+  i18n <- Translator$new(translation_json_path = "data/translation.json")
+  i18n$set_translation_language("en")
+  # Act
+  i18ntag <- usei18n(i18n)
+  # Assert
+  expect_equal(
+    class(i18ntag),
+    c("shiny.tag.list", "list")
+  )
+  expect_equal(length(i18ntag[[1]]$children), 2)
```

Comment:
You are checking only the length. It would be great to also check the content. The first child should include translations, second should be the js file.

Reply:
I tried doing this with `shin.tag` with `tags$script` but couldn't make `expect_equal` work. Because of this, I set them as characters